stumpy/gpu_stump.py

```diff
@@ -154,23 +155,23 @@ def _compute_and_update_PI_kernel(
 
 
 def _gpu_stump(
-    T_A,
-    T_B,
-    m,
-    range_stop,
-    excl_zone,
-    M_T,
-    Σ_T,
-    QT,
-    QT_first,
-    μ_Q,
-    σ_Q,
-    k,
-    ignore_trivial=True,
-    range_start=1,
-    threads_per_block=THREADS_PER_BLOCK,
-    device_id=0,
-):
+    T_A: np.ndarray,
+    T_B: np.ndarray,
+    m: int,
+    range_stop: int,
+    excl_zone: int,
+    M_T: np.ndarray,
+    Σ_T: np.ndarray,
+    QT: np.ndarray,
+    QT_first: np.ndarray,
+    μ_Q: np.ndarray,
+    σ_Q: np.ndarray,
+    k: int,
+    ignore_trivial: bool = True,
+    range_start: int = 1,
+    threads_per_block: int = THREADS_PER_BLOCK,
+    device_id: int = 0,
```

Comment:
I have some yet-to-be-committed updates where `device_id` can either be a single `int` (as it is shown here) or a list of `int`. Can you please tell me what this line should be if `device_id` can be either a single `int` or a list of `int`?  This way, I can update this on my own after your PR.

Reply:
device_id: Union[int, List[int]] = 0
I think. Union => https://mypy.readthedocs.io/en/latest/kinds_of_types.html#union-types